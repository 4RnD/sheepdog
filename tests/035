#!/bin/bash

# Test sockfd keepalive

seq=`basename $0`
echo "QA output created by $seq"

here=`pwd`
tmp=/tmp/$$
status=1        # failure is the default!

trap "_uninit; exit \$status" 0 1 2 3 15

# get standard environment, filters and checks
. ./common.rc
. ./common.filter

_uninit()
{
	for i in `seq 1 4`; do
		_cleanup_machine_simulation $i
	done
}

_cleanup

for i in `seq 0 5`; do
	_start_sheep $i
done

_wait_for_sheep 6

$COLLIE cluster format -c 3 -m unsafe

$COLLIE vdi create test 40M
dd if=/dev/urandom | $COLLIE vdi write test &

sleep 3
for i in `seq 1 4`; do
	_simulate_machine_down $i
done

_wait_for_collie

for i in `seq 0 9`; do
	$COLLIE vdi object -i $i test
done

status=0
