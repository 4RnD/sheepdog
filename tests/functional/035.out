QA output created by 035
using backend plain store
Looking for the object 0x7c2b2500000000 (vid 0x7c2b25 idx 0, 6 copies) with 6 nodes

127.0.0.1:7000 has the object
127.0.0.1:7005 has the object
127.0.0.1:7006 has the object
127.0.0.1:7007 has the object
127.0.0.1:7008 has the object
127.0.0.1:7009 has the object

According to sheepdog algorithm, the object should be located at:
127.0.0.1:7008 127.0.0.1:7009 127.0.0.1:7006 127.0.0.1:7007 127.0.0.1:7000 127.0.0.1:7005
Looking for the object 0x7c2b2500000001 (vid 0x7c2b25 idx 1, 6 copies) with 6 nodes

127.0.0.1:7000 has the object
127.0.0.1:7005 has the object
127.0.0.1:7006 has the object
127.0.0.1:7007 has the object
127.0.0.1:7008 has the object
127.0.0.1:7009 has the object

According to sheepdog algorithm, the object should be located at:
127.0.0.1:7007 127.0.0.1:7000 127.0.0.1:7006 127.0.0.1:7005 127.0.0.1:7008 127.0.0.1:7009
Looking for the object 0x7c2b2500000002 (vid 0x7c2b25 idx 2, 6 copies) with 6 nodes

127.0.0.1:7000 has the object
127.0.0.1:7005 has the object
127.0.0.1:7006 has the object
127.0.0.1:7007 has the object
127.0.0.1:7008 has the object
127.0.0.1:7009 has the object

According to sheepdog algorithm, the object should be located at:
127.0.0.1:7006 127.0.0.1:7000 127.0.0.1:7005 127.0.0.1:7009 127.0.0.1:7007 127.0.0.1:7008
Looking for the object 0x7c2b2500000003 (vid 0x7c2b25 idx 3, 6 copies) with 6 nodes

127.0.0.1:7000 has the object
127.0.0.1:7005 has the object
127.0.0.1:7006 has the object
127.0.0.1:7007 has the object
127.0.0.1:7008 has the object
127.0.0.1:7009 has the object

According to sheepdog algorithm, the object should be located at:
127.0.0.1:7006 127.0.0.1:7008 127.0.0.1:7005 127.0.0.1:7007 127.0.0.1:7009 127.0.0.1:7000
Looking for the object 0x7c2b2500000004 (vid 0x7c2b25 idx 4, 6 copies) with 6 nodes

127.0.0.1:7000 has the object
127.0.0.1:7005 has the object
127.0.0.1:7006 has the object
127.0.0.1:7007 has the object
127.0.0.1:7008 has the object
127.0.0.1:7009 has the object

According to sheepdog algorithm, the object should be located at:
127.0.0.1:7005 127.0.0.1:7006 127.0.0.1:7000 127.0.0.1:7008 127.0.0.1:7009 127.0.0.1:7007
Looking for the object 0x7c2b2500000005 (vid 0x7c2b25 idx 5, 6 copies) with 6 nodes

127.0.0.1:7000 has the object
127.0.0.1:7005 has the object
127.0.0.1:7006 has the object
127.0.0.1:7007 has the object
127.0.0.1:7008 has the object
127.0.0.1:7009 has the object

According to sheepdog algorithm, the object should be located at:
127.0.0.1:7009 127.0.0.1:7007 127.0.0.1:7005 127.0.0.1:7006 127.0.0.1:7008 127.0.0.1:7000
Looking for the object 0x7c2b2500000006 (vid 0x7c2b25 idx 6, 6 copies) with 6 nodes

127.0.0.1:7000 has the object
127.0.0.1:7005 has the object
127.0.0.1:7006 has the object
127.0.0.1:7007 has the object
127.0.0.1:7008 has the object
127.0.0.1:7009 has the object

According to sheepdog algorithm, the object should be located at:
127.0.0.1:7000 127.0.0.1:7007 127.0.0.1:7005 127.0.0.1:7006 127.0.0.1:7008 127.0.0.1:7009
Looking for the object 0x7c2b2500000007 (vid 0x7c2b25 idx 7, 6 copies) with 6 nodes

127.0.0.1:7000 has the object
127.0.0.1:7005 has the object
127.0.0.1:7006 has the object
127.0.0.1:7007 has the object
127.0.0.1:7008 has the object
127.0.0.1:7009 has the object

According to sheepdog algorithm, the object should be located at:
127.0.0.1:7008 127.0.0.1:7006 127.0.0.1:7005 127.0.0.1:7000 127.0.0.1:7009 127.0.0.1:7007
Looking for the object 0x7c2b2500000008 (vid 0x7c2b25 idx 8, 6 copies) with 6 nodes

127.0.0.1:7000 has the object
127.0.0.1:7005 has the object
127.0.0.1:7006 has the object
127.0.0.1:7007 has the object
127.0.0.1:7008 has the object
127.0.0.1:7009 has the object

According to sheepdog algorithm, the object should be located at:
127.0.0.1:7006 127.0.0.1:7005 127.0.0.1:7000 127.0.0.1:7008 127.0.0.1:7009 127.0.0.1:7007
Looking for the object 0x7c2b2500000009 (vid 0x7c2b25 idx 9, 6 copies) with 6 nodes

127.0.0.1:7000 has the object
127.0.0.1:7005 has the object
127.0.0.1:7006 has the object
127.0.0.1:7007 has the object
127.0.0.1:7008 has the object
127.0.0.1:7009 has the object

According to sheepdog algorithm, the object should be located at:
127.0.0.1:7008 127.0.0.1:7005 127.0.0.1:7006 127.0.0.1:7007 127.0.0.1:7009 127.0.0.1:7000
